src/app/[locale]/projects/add/page.tsx

```diff
@@ -7,456 +7,133 @@
 // SPDX-License-Identifier: EPL-2.0
 // License-Filename: LICENSE
 
-'use client'
+"use client"
 
-import styles from '@/css/AddProjects.module.css'
-import AddKeyValueComponent from '@/components/AddKeyValue'
-import SelectCountryComponent from '@/components/SelectCountry'
-import AddAdditionalRolesComponent from '@/components/AddAdditionalRoles'
-import { SearchUsersModal } from '@/components/sw360'
-import Link from 'next/link'
-import { usePathname } from 'next/navigation'
+import { Col, Row, ListGroup, Tab, Button } from 'react-bootstrap'
+import Summary from "@/components/ProjectAddSummary/Summary"
+import { useTranslations } from 'next-intl'
+import { COMMON_NAMESPACE } from '@/object-types/Constants'
+import { useRouter } from 'next/navigation'
+import HttpStatus from '@/object-types/enums/HttpStatus'
+import { TypeOptions, toast } from 'react-toastify'
+import ApiUtils from '@/utils/api/api.util'
+import ProjectPayload from "@/object-types/CreateProjectPayload"
+import { useState } from 'react'
+import { AUTH_TOKEN } from '@/utils/env'
 
-export default function Summary() {
-    const pathName = usePathname()
+
+export default function AddProjects() {
+
+    const router = useRouter();
+    const t = useTranslations(COMMON_NAMESPACE)
+    const [projectPayload, setProjectPayload] = useState<ProjectPayload>({
+        name: '',
+        description: '',
+        version: '',
+        visibility: '',
```

Comment:
**visibility** and **projectType** should have default value (default option on GUI).

Reply:
Resolved ! 👍